lib/swing-lib/src/main/java/org/triplea/swing/jpanel/GridBagConstraintsBuilder.java

```diff
@@ -26,8 +26,8 @@
  * </pre></code>
  */
 public class GridBagConstraintsBuilder {
-  private final int gridX;
-  private final int gridY;
+  private int gridX;
+  private int gridY;
```

Comment:
If these values are no longer required (via constructor); adding a no-args constructor would be consistent with the other builders.